core/service/src/client/tests/centralized/mod.rs

```diff
@@ -6,6 +6,6 @@ mod misc_tests;
 #[cfg(feature = "slow_tests")]
 mod nightly_tests;
 mod public_decryption_tests;
-#[cfg(feature = "insecure")]
+#[cfg(feature = "slow_tests")]
```

Comment:
I am wondering why there are no complication issues when just compiling with `slow_tests` and not also `insecure` since the test code use code filtered by `insecure`